test/e2e/helpers.js

```diff
@@ -13,7 +13,7 @@ const tinyDelayMs = 200;
 const regularDelayMs = tinyDelayMs * 2;
 const largeDelayMs = regularDelayMs * 2;
 const veryLargeDelayMs = largeDelayMs * 2;
-const dappPort = 8080;
+const dappBasePort = 808;
```

Comment:
I think it could be a little confusing to see the number `808` as a base port, because that is later coerced to a string and concatenated with another string. And this causes the the default and most common case (8080) to never be explicitly written in the code. Maybe setting it to the number `8080` and then using arithmetic to set the port below (e.g. dappServer[i].listen(`${dappBasePort + i`);`

Reply:
sure, I've made a few tweaks, so we are now utilizing the same pattern (using `dappOptions`, like `ganacheOptions`) @PeterYinusa

Reply:
Oops, I hadn't realised you had already made the arithmetic change. I was suggesting using the same pattern as ganacheOptions too explicitly specify the port of the second instance as an alternative to the loop and port calculations. Sorry for unnecessary extra work proposed

Reply:
ohh no problem! I guess the new changes also simplify a bit the approach :) thanks!